pkg/noderesourcetopology/score.go

```diff
@@ -0,0 +1,141 @@
+package noderesourcetopology
+
+import (
+	"context"
+	"fmt"
+
+	topologyv1alpha1 "github.com/k8stopologyawareschedwg/noderesourcetopology-api/pkg/apis/topology/v1alpha1"
+	v1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/resource"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/klog/v2"
+	framework "k8s.io/kubernetes/pkg/scheduler/framework/v1alpha1"
+	apiconfig "sigs.k8s.io/scheduler-plugins/pkg/apis/config"
+)
+
+const (
+	// Name is the name of the plugin used in the plugin registry and configurations.
+	ScorerName = "NumaNodeScorer"
```

Comment:
since it's implementation for least allocatable score plugin and we're planning to add another score plugins,  I suggest to name it in little bit another way. 
score.go - least_allocatable_score.go
ScorerName - LeastAllocatableName = "NodeResourceTopologyLeastAllocatableScore"